release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -347,11 +347,18 @@ You can now remove or rotate the GCP admin-level credential that the xref:../ope
 
 You can now use the Compliance Operator to perform Center for Internet Security (CIS) Kubernetes Benchmark checks. CIS profiles for {product-title} are based on the CIS Kubernetes checks.
 
+Until the CIS {product-title} Benchmark is published, you can refer to the link:https://pnt.redhat.com/pnt/p-18428165/Red_Hat_OpenS...uide_v1.0.pdf[Red Hat {product-title} Hardening Guide].
+
 [id="ocp-4-7-security-secure-boot"]
 ==== Secure Boot support for installer-provisioned clusters
 
 You can now deploy a cluster with Secure Boot when using installer-provisioned infrastructure on bare metal nodes. Deploying a cluster with Secure Boot requires UEFI boot mode and Red Fish Virtual Media. You cannot use self-generated keys with Secure Boot.
 
+[id="ocp-4-7-cmp-acm-2-2"]
+==== Advanced Cluster Management 2.2 integration
```

Comment:
Addressing point 3 of https://github.com/openshift/openshift-docs/issues/26801#issuecomment-766500209
@xiaojiey